mujoco_warp/_src/collision_primitive.py

```diff
@@ -194,1325 +232,349 @@ def write_contact(
 
 
 @wp.func
-def _plane_sphere(plane_normal: wp.vec3, plane_pos: wp.vec3, sphere_pos: wp.vec3, sphere_radius: float):
-  dist = wp.dot(sphere_pos - plane_pos, plane_normal) - sphere_radius
-  pos = sphere_pos - plane_normal * (sphere_radius + 0.5 * dist)
-  return dist, pos
-
-
-@wp.func
-def plane_sphere(
+def plane_sphere_wrapper(
```

Comment:
this is not an api function, so maybe `_plane_sphere_wrapper`? (similar comment for all the wrappers in this file)